bin/rbenv-install

```diff
@@ -7,7 +7,8 @@
 #        rbenv install -l|--list
 #        rbenv install --version
 #
-#   -l/--list          List all available versions
+#   -l/--list          List latest stable versions for each Ruby
+#   -L/--list-all      List all available versions
```

Comment:
Looks like the /available/local/ replacement got lost in the merge.

(also applies to the help message printed at the end of the `--list` output below)

Reply:
Oops. Let me some time to fix it. I'll fix it tomorrow.

Reply:
Actually, it wasn't changed from the beginning. I forgot to change `bin/rbenv-install` in #1419.

Reply:
Fixed.